dotnet/Capstone/DAO/BirdSightingSqlDao.cs

```diff
@@ -0,0 +1,179 @@
+﻿using Capstone.Exceptions;
+using Capstone.Models;
+using System;
+using System.Collections.Generic;
+using System.Data.SqlClient;
+using System.Xml.Linq;
+
+namespace Capstone.DAO
+{
+    public class BirdSightingSqlDao : IBirdSightingDao
+
+    {
+        private readonly string connectionString;
+
+        private readonly IUserDao userDao;
+
+        public BirdSightingSqlDao(string connectionString)
+        {
+            this.connectionString = connectionString;
+            this.userDao = new UserSqlDao(connectionString);
+        }
+
+
+        public BirdSighting addSighting(BirdSighting birdSighting, string username)
```

Comment:
It would be good to add in a guard clause either here or in the controller to prevent adding a sighting on a future date. If that doesn't happen this PR, I'm not worried about it.